### ISSUE TITLE
HTTP connector documentation missing

### ISSUE THREAD

**Issue by delfuego** [MicrosoftDocs/feedback#2521]
**Link to article:**
https://docs.microsoft.com/en-us/connectors/

**Problem:**
The documentation for the built-in HTTP connector (for Power Automate, Power Apps, and Logic Apps) is missing entirely from the site. Similar connectors (e.g., FTP) are fully-documented, but HTTP is... just missing.

**Comment by rootsmusic** [comment]
@delfuego Do you mean HTTP with Azure AD at https://docs.microsoft.com/en-us/connectors/webcontents/ ?

**Comment by delfuego** [comment]
@rootsmusic No, a year-plus ago, I meant the straight, plain "HTTP" connector… which now, a year-plus later, appears to have disappeared, at least in our tenant. And was replaced by the "HTTP with Azure AD" connector, which (of course) is a premium connector... meaning that users no longer have any way to just make a straight HTTP call. *sigh*

**Comment by rootsmusic** [comment]
@delfuego You can complain by clicking on the button at the bottom of https://docs.microsoft.com/en-us/connectors/webcontents/

**Comment by rootsmusic** [comment]
(Sorry @delfuego, the doc's author isn't the connector's developer.)  You can suggest a connector on the product's ideas forum.
